README.md

```diff
@@ -1,29 +1,61 @@
-# AMDGPUnative
+# AMDGPUnative.jl
 
-*Support for compiling and executing native Julia kernels on AMD hardware.*
+*Support for compiling and executing native Julia kernels on AMD GPUs.*
 
-NOTE:
------
+| **Documentation**                                                       | **Build Status**                                              |
+|:---------------------------------------:|:-------------------------------------------------------------:|
+| [![][docs-master-img]][docs-master-url] | [![][gitlab-img]][gitlab-url] [![][codecov-img]][codecov-url] |
 
-This is an experimental package, not ready for usage and subject to change at any time. Contributions are welcomed, but here be dragons.
+[gitlab-img]: https://gitlab.com/JuliaGPU/AMDGPUnative.jl/badges/master/pipeline.svg
+[gitlab-url]: https://gitlab.com/JuliaGPU/AMDGPUnative.jl/commits/master
 
-Installation
-------------
+[codecov-img]: https://codecov.io/gh/JuliaGPU/AMDGPUnative.jl/branch/master/graph/badge.svg
+[codecov-url]: https://codecov.io/gh/JuliaGPU/AMDGPUnative.jl
 
-AMDGPUnative is a experimental package, to install it use `Pkg.clone`.
+[docs-master-img]: https://img.shields.io/badge/docs-master-blue.svg
+[docs-master-url]: https://juliagpu.gitlab.io/AMDGPUnative.jl/
 
-It is recommended to use LLVM 7.0 or higher, otherwise certain features (like
-math intrinsics) are disabled.
 
-Acknowledgment
---------------
 
-AMDGPUnative would not have been possible without the work by Tim Besard on [CUDAnative.jl](https://github.com/JuliaGPU/CUDAnative.jl)
-and [LLVM.jl](https://github.com/maleadt/LLVM.jl). 
+## Quick start
 
-In revision to: commit 53966d812d63c7cbea0de114ebee4033c7ddad0d of CUDAnative.jl 
-License
--------
+The package can be installed with the Julia package manager.
+From the Julia REPL, type `]` to enter the Pkg REPL mode and run:
 
-AMDGPUnative.jl is licensed under the [MIT License](LICENSE.md).
+```
+pkg> add AMDGPUnative
+```
+
+Or, equivalently, via the `Pkg` API:
+
+```julia
+julia> import Pkg; Pkg.add("AMDGPUnative")
+```
+
+
+## Project Status
+
+The package is tested against, and being developed for, Julia `1.3` and above.
+Only 64-bit Linux is supported and working at this time, until ROCm is ported
+to other platforms.
+
+It is recommended to use a version of Julia with LLVM 9.0 or higher.
```

Comment:
Could we add a statement of the experimental nature of this package since there's a couple of show-stopper issues (e.g. #58, no stable array interface, etc.)? Or even better, add a table of implemented functionality that we can update as the features are implemented, for instance:

```suggestion
It is recommended to use a version of Julia with LLVM 9.0 or higher.

### Supported Functionality

| Feature | Supported | Notes |
|:---|:---:|:---|
| Host-side kernel launches | :heavy_check_mark: | See #58 |
| Dynamic parallelism | :x: |
| Local (shared) memory | :x: |
| Coarse-grained memory | :x: |
| Page-locked (pinned) memory | :x: |
```
to get:

| Feature | Supported | Notes |
|:---|:---:|:---|
| Host-side kernel launches | :heavy_check_mark: | See #58 |
| Dynamic parallelism | :x: |
| Local (shared) memory | :x: |
| Coarse-grained memory | :x: |
| Page-locked (pinned) memory | :x: |